src/package/utils/windowMock/index.js

```diff
@@ -0,0 +1,13 @@
+const windowMock = {
+  location: {
+    href: '',
+    reload: () => null,
+  },
+  localStorage: {
+    setItem: () => null,
+    getItem: () => null,
+    removeItem: () => null,
+  },
+}
+
+export default windowMock
```

Comment:
Was going to use a library for this, but they were all really heavy or had global side effects; this is simpler.